tests/acceptance/tests/Visual/Administration/Promotions.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { test, expect, setViewport, replaceElements } from '@fixtures/AcceptanceTest';
+
+test('Visual: Promotions Listing Page', { tag: '@Visual' }, async ({ 
+    ShopAdmin, 
+    TestDataService, 
+    AdminPromotionsListing,
+ }) => {
+    await test.step('Creates a screenshot of the promotions listing page in its empty state.', async () => { 
+        await ShopAdmin.goesTo(AdminPromotionsListing.url());
+        await setViewport(AdminPromotionsListing.page, {
+            width: 1440,
+            contentHeight: 1200,
+        });
+        await expect(AdminPromotionsListing.page.locator('.sw-desktop__content')).toHaveScreenshot('Listing-Empty-State.png');
+    });
+
+    await test.step('Creates a screenshot of the promotions listing page with an active and an inactive promotion.', async () => {
+        await TestDataService.createPromotionWithCode({ active: true });
+        await TestDataService.createPromotionWithCode({ active: false });
+        
+        await ShopAdmin.goesTo(AdminPromotionsListing.url());
+        await replaceElements(AdminPromotionsListing.page, [ '.sw-data-grid__cell--name' ]);
+        await setViewport(AdminPromotionsListing.page, {
+            width: 1440,
+            contentHeight: 1200,
+        }); 
+        await expect(AdminPromotionsListing.page.locator('.sw-desktop__content')).toHaveScreenshot('Listing-With-Promotions.png'); 
```

Comment:
This should also be a locator. I have been using `contentView` as the locator name for these.